languages/csharp/exercises/concept/squeaky-clean/.meta/design.md

```diff
@@ -25,5 +25,5 @@
 ## Prerequisites
 
 - `strings`: know of the `string` type that will be iterated over and accessed by index.
-- `for-loop` for loops (rather than foreach) are the best means of highlighting the relationship between strings and `char`s
-- `if-statements`:
+- `for-loop`: for loops (rather than foreach) are the best means of highlighting the relationship between strings and `char`s
```

Comment:
Just one thing I noticed while reviewing, this concept should be `for-loops`. We should fix this in a separate PR.